docs/lib/project-setup/fragments/android/project-setup/50_provisionBackend.md

```diff
@@ -0,0 +1,22 @@
+### Step 4: Provision the backend with Amplify CLI
```

Comment:
I think we move this up. A user might want to start with storage, in which case the IDE integrations are insufficient. I can deal with this is my pass.

Reply:
Currently the order is
1. create project
2. add dependencies (cocoapods/gradle)
3. verify dependencies were added
4. provision backend
5. verify
6. next steps.

How are you proposing to rearrange this?

Reply:
Not going to update this in this PR